react/react1/week2/vite-project/src/search.jsx

```diff
@@ -0,0 +1,17 @@
+import { useRef } from "react";
+
+const Search = ({ filterList }) => {
+  const input = useRef(null);
+  return (
+    <div className="search-input">
+      <input
+        placeholder="search"
+        onChange={() => filterList(input.current.value)}
+        type="search"
+        ref={input}
+      ></input>
```

Comment:
you should be able to just use your closing tag within the first input (<input />) when you don't have any child elements..